src/jquery.mask.js

```diff
@@ -546,6 +549,7 @@
         clearInterval($.maskWatchers[this.selector]);
         delete $.maskWatchers[this.selector];
         return this.each(function() {
+			$(this).removeAttr('placeholder');
```

Comment:
@PauloHDSousa we only want to remove the placeholder if we were the one who set it. It's that simple.